scripts/generate_dump

```diff
@@ -698,16 +1003,25 @@ main() {
     save_cmd "sysctl -a" "sysctl"
 
     save_ip_info
+
+    save_bridge "fdb show" "fdb"
+    save_bridge "vlan show" "vlan"
```

Comment:
How about rename API to 'save_bridge_info' and move these two to that API. In future all l2 info collection can be put in there.

Reply:
Done